feature/amazon/src/main/java/com/revenuecat/purchases/amazon/AmazonCache.kt

```diff
@@ -17,7 +17,7 @@ internal class AmazonCache(
     }
 
     @Synchronized
-    fun setReceiptSkus(receiptsToSkus: Map<String, String>) {
+    fun cacheReceiptSkus(receiptsToSkus: Map<String, String>) {
```

Comment:
thoughts on renaming to `cacheReceiptSkusByToken`? i kinda like our ios naming rule on dictionaries `valuesByKey`